### PR TITLE
update bridges section & chainbridge page

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the Builders > Integrations > Bridges section and the ChainBridge page itself. It takes care of the following items:

- get rid of eth directory
- add intro diagram
- number & reorder images/diagrams
- general update to chainbridge docs

⭕️ I'll circle back to the checklist. Hoping to put a script together to move pages so that can be used to test & update the Chinese site with. ⭕️

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If images have been moved, I have created an additional PR in `moonbeam-docs-cn` to update images to their new paths
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

https://github.com/PureStake/moonbeam-mkdocs/pull/78 -- ⭕️ Will need to circle back & update the CN mkdocs.yml file ⭕️

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Review by albertov19 — CHANGES_REQUESTED** [review]
Thanks for fixing a bunch of styling issues this page had.  Left some minor feedback

File: builders/integrations/bridges/chainbridge.md [hunk 1]
```diff
@@ -5,66 +5,61 @@ description: How to use ChainBridge to connect and transfer assets such as ERC-2
...
 
-This guide is broken down into two main sections. In the first part, we'll explain the general workflow of the bridge. In the second part, we'll go through a couple of examples using the bridge to transfer ERC-20 and ERC-721 assets between Moonbase Alpha and Kovan/Rinkeby. 
+This guide is broken down into two main sections. In the first part, the general workflow of the bridge will be explained. In the second part, there will be a couple of examples using the bridge to transfer ERC-20 and ERC-721 assets between Moonbase Alpha and Kovan/Rinkeby. 
 
- - [How the bridge works](/builders/integrations/bridges/eth/chainbridge/#how-the-bridge-works)
```
[the remark this answers is not in the document]
Comment (eshaben) [reply builders/integrations/bridges/chainbridge.md:16]: yeah that is why i removed it

**Comment by eshaben** [comment]
@albertov19 I think it's difficult to put a solution in for ChainBridge that would apply for other bridging solutions since the bridge only goes between Moonbase Alpha and two other TestNets

I thought about a tabbed solution with a table in it, but it seems a bit much for this instance. Also not sure if it even makes sense to have a table in that scenario since `Chain A` would always be whatever network the tab is on 🤷‍♀️ 🤷‍♀️

**Comment by albertov19** [comment]
I think probably a tabbed table/list is the way to go. Even though it seems overkill for Chainbridge because it only connects Moonbase Alpha with Kovan/Rinkeby. For example, if another bridge connects Moonbase Alpha with other TestNets but also Moonbeam with 4 more networks you'll have 2 tabs with quite a bit of info in it. 

Not sure what is the best approach, but just an idea

**Comment by eshaben** [comment]
right yeah I figured it would be fine for more robust bridges but was struggling with the "overkill" of it in this instance. also struggling to make it even look good because a list in the tab doesn't look good, and the one column table isn't much better but it works 😬

**Review by albertov19 — APPROVED** [review]
LGTM 🚀

**Pull request merged**